<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab2SocketClient</class>
 <widget class="QWidget" name="Tab2SocketClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,2,2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>수신 데이터</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pPBrecvDataClear">
         <property name="text">
          <string>수신 삭제</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pPBServerConnect">
         <property name="text">
          <string>서버 연결</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextEdit" name="pTErecvData">
         <property name="font">
          <font>
           <family>Georgia</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,6,2">
       <item>
        <widget class="QLineEdit" name="pLErecvId"/>
       </item>
       <item>
        <widget class="QLineEdit" name="pLEsendData"/>
       </item>
       <item>
        <widget class="QPushButton" name="pPBsendButton">
         <property name="text">
          <string>전송</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
